pkg/fileservice/mem_cache.go

```diff
@@ -65,20 +85,18 @@ func NewMemCache(
 		}
 	}
 
-	capacityFunc := func() int64 {
-		// read from global hint
-		if n := GlobalMemoryCacheSizeHint.Load(); n > 0 {
-			return n
-		}
-		// fallback
-		return capacity()
-	}
 	dataCache := fifocache.NewDataCache(capacityFunc, postSetFn, postGetFn, postEvictFn)
 
-	return &MemCache{
+	ret := &MemCache{
 		cache:       dataCache,
 		counterSets: counterSets,
 	}
+
+	if name != "" {
```

Comment:
fix for `handleFileServiceCacheEvictRequest`, @reusee  ptal

Reply:
LGTM